translations/catroweb.en.yml

```diff
@@ -205,6 +205,11 @@ programs:
   confrimDelete: Are you sure you want to delete the translation?
   default: Default
   save: Save
+  translationTitle: '%language% Translation'
```

Comment:
translation*

We try to keep the lower/uppercases grammatically correct.

Reply:
Updated to "translation"